yao/roster.json

```diff
@@ -254,7 +254,12 @@
       "github": "EndlessEndless1999",
       "materials": [{
         "type": "cv"
-      }]
+      },
+      {
+        "type": "video",
+        "url": "https://www.youtube.com/watch?v=6Ack9kf1JAE"
+      }
```

Comment:
Can you please update your object, and make it look like this

```
 {
        "type": "video",
        "url": "6Ack9kf1JAE"
      }
```

Reply:
should be good now